aws/efs/README.md

```diff
@@ -5,9 +5,109 @@
 quay.io/external_storage/efs-provisioner:v0.1.0
 ```
 
+The efs-provisioner is a storage plugin for kubernetes. It consists of a container that has access to an AWS [EFS](https://aws.amazon.com/efs/) resource. The container reads a configmap which contains the EFS filesystem ID, the AWS region and the name you want to use for your efs-provisioner. This name will be used later when you create a storage class.
+
 ## Prerequisites
 * An EFS file system in your cluster's region
 * [Mount targets](http://docs.aws.amazon.com/efs/latest/ug/accessing-fs.html) and [security groups](http://docs.aws.amazon.com/efs/latest/ug/accessing-fs-create-security-groups.html) such that any node (in any zone in the cluster's region) can mount the EFS file system by its [File system DNS name](http://docs.aws.amazon.com/efs/latest/ug/mounting-fs-mount-cmd-dns-name.html)
+* A directory on the EFS. Can be anything but in the guide we will be using `/persistentvolumes`.
+
+## Getting Started
+
+If you are new to Kubernetes or to PersistentVolumes this quick start will get you up and running with simple defaults.
+
+- Download the manifest file [manifest.yaml](deploy/manifest.yaml).
+  ```
+  wget https://raw.githubusercontent.com/kubernetes-incubator/external-storage/master/aws/efs/deploy/manifest.yaml
+  ```
+
+- In the configmap section change the `file.system.id:` and `aws.region:` to match the details of the EFS you created.
+
+- In the deployment section change the `server:` to the DNS endpoint of the EFS you created.
+
+- `kubectl apply -f manifest.yaml` 
+
+- Now you can use your PersistentVolume in your pod or deployment by referencing your claim name when it's created.
+
+```
+kind: Pod
+apiVersion: v1
+metadata:
+  name: test-pod
+spec:
+  containers:
+  - name: test-pod
+    image: gcr.io/google_containers/busybox:1.24
+    command:
+      - "/bin/sh"
+    args:
+      - "-c"
+      - "touch /mnt/SUCCESS && exit 0 || exit 1"
+    volumeMounts:
+      - name: efs-pvc
+        mountPath: "/mnt"
+  restartPolicy: "Never"
+  volumes:
+    - name: efs-pvc
+      persistentVolumeClaim:
+        claimName: efs
+```
+
+Because our claim is ReadWriteMany if you scale this pod each aditional pod will also be able to read and write the same files. You may also reference the same claimName in another type of pod so your 2 applications can read and write the same files. If you wish to have a second application that uses EFS storage but don't want other pods to access the files, create a new claim using a new name but the same storage class.
```

Comment:
I would just remove this 'Because our claim is ReadWriteMany...', see below comments